contrib/bmp2/README.md

```diff
@@ -0,0 +1,38 @@
+BMP2
+====
+
+![BMP2](http://ebrombaugh.studionebula.com/embedded/bmp2/bmp2.jpg)
```

Comment:
Is it possible for you to add these assets in the repository? I think it would be better not to link to an external website here and keep the assets in the repository. Maybe ask @emeb if that would be ok with him?

Reply:
That would be great. Maybe also change the name to something that fits with your plans better?